docs/sources/flow/reference/cli/run.md

```diff
@@ -63,26 +63,61 @@ reloading.
 
 [component controller]: {{< relref "../../concepts/component_controller.md" >}}
 
-## Clustered mode (experimental)
+## Clustering (beta)
 
-When the `--cluster.enabled` command-line argument is provided, Grafana Agent will
-start in _clustered mode_.
+The `--cluster.enabled` command-line argument starts Grafana Agent in
+[clustering][] mode. The rest of the `--cluster.*` command-line flags can be
+used to configure how nodes discover and connect to one another.
 
-The agent tries to connect over HTTP/2 to one or more peers provided in the
-comma-separated `--cluster.join-addresses` list to join an existing cluster.
-If no connection can be made or the argument is empty, the agent falls back to
-bootstrapping a new cluster of its own.
+Each cluster member’s name must be unique within the cluster. Nodes which try
+to join with a conflicting name are rejected and will fall back to
+bootstrapping a new cluster of their own.
 
-Each node's name must be unique within the cluster. The node name defaults to
-the machine's hostname but can be set to a different value by using the
-`--cluster.node-name` flag. Mainly useful for running clustered agents on the
-same machine.
+Peers communicate over HTTP/2 on the agent's built-in HTTP server. Each node
+must be configured to accept connections on `--server.http.listen-addr` and the
+address defined or inferred in `--cluster.advertise-address`.
 
-The agent will advertise its own address as `--cluster.advertise-address` to
-other agent nodes; if this is empty it will attempt to find a suitable address
-to advertise from a list of default network interfaces. The agent must be
-reachable over HTTP on this address as communication happens over the agent's
-HTTP server.
+If the `--cluster.advertise-address` flag is not explicitly set, the agent
+tries to infer a suitable one from the `eth0` and `en0` local network
```

Comment:
`eth0` and `en0` are Linux-specific terms. Maybe someone will think it doesn't work on windows?

Reply:
That's correct, eth0 and en0 are always used as the defaults meaning it won't work on Windows. I've clarified that Windows users need to provide a value for this flag here.